exercises/ex4-functions/functions/functions_test.go

```diff
@@ -1,4 +1,4 @@
-package functions_test
+package functions
```

Comment:
BTW, files in folder named as `testdata` or files with `_test.go` suffix will not be build by a regular `go build`. 
They will be build only during `go test`.

Reply:
Oh, this got changed when moving files around.